### PR TITLE
Refactor ( Polymer 1.0 )

### PR DESCRIPTION
**Pull request by kdzwinel**
I picked up @ericguzman work on "polymerizing" this app and finished the refactor.

### CODE REVIEW

**Comment by paulirish** [comment]
thanks for this awesome work. and for the polymer 1.0!

now to make some upgrades. :)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Unix-style newlines with a newline ending every file
+# UTF-8 - default charset
+# 2 space indentation
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = space
+indent_size = 2
```
Comment (arthurvr) [inline .editorconfig]: this file is here to ensure final newlines lol :p

Comment (kdzwinel) [inline .editorconfig]: haha, good catch

Comment (arthurvr) [inline .editorconfig]: :p

**Pull request merged**